### PR TITLE
Mark Item as Purchased

### PR DESCRIPTION
**Pull request by natush2**

### CODE REVIEW

File: components/ShoppingItemForm/ShoppingItemForm.js
Comment (rvoll) [inline components/ShoppingItemForm/ShoppingItemForm.js:1]: Looks good. No comments. :)

File: components/ItemsList/StyledItemsList.js
Comment (rvoll) [inline components/ItemsList/StyledItemsList.js:1]: Great. I din't find any problematic issues.

**Review by rvoll — COMMENTED** [review]
Hi Nataliia, I'm very impressed and I learn a lot from your PR. I will stop for now as someone else is already checking it and I don't have time this afternoon. Great job! :)

File: components/Header/Header.js
[the remark this answers is not in the document]
Comment (natush2) [reply components/Header/Header.js:1]: Thank you for your feedback, I appreciate it. Thank you again ❤

File: components/Item/Item.js [hunk 1]
```diff
@@ -69,6 +81,18 @@ export default function Item({
             <p>Are you sure you want to delete this item?</p>
           </ModalDelete>
         )}
+
+        <StyledrStatusWrrepe>
+          <StyledrStatusText>
+            {isPurchasedView
+              ? "Move the item to the purchased list."
+              : "Move the item back to the shopping list."}
```
[the remark this answers is not in the document]
Comment (natush2) [reply components/Item/Item.js:89]: Yeah, I noticed that too 😅

**Review by NikRoe — APPROVED** [review]
LGTM

**Pull request merged**